src/components/Navbar.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import { NavLink, Link } from 'react-router-dom';
+import logo from '../assets/logo.png';
+
+function Navbar() {
+  const [links] = useState([
+    { path: '/', text: 'Home' },
+    { path: 'category', text: 'Category' },
+  ]);
+  return (
+    <nav>
+      <Link to="/">
+        <img src={logo} id="logo" alt="Logo" />
+      </Link>
+      <ul className="nav-items">
+        {links.map((link) => <li key={link.text} className="nav-item"><NavLink to={link.path}>{link.text}</NavLink></li>)}
```

Comment:
- [ ] **[OPTIONAL]** Great job using a links array to handle your navigation links, this is a highly scalable way of structuring your code for possible future changes, kudos for this! 🚀 However, as this is not intended to be modified within your code, I would suggest not using `useState` for this and defining a constant instead, like this: 🤓 🔧 

```js
const LINKS = [{ path: '/', text: 'Home' }, { path: 'category', text: 'Category' }]
```